docker/packager/binary/build.sh

```diff
@@ -55,6 +55,17 @@ ccache_status
 # clear cache stats
 ccache --zero-stats ||:
 
+# Check if there are some additional scripts we would need to run
```

Comment:
It looks like pre-build scripts to me. So in case of post-builds, we'd need to touch it one more time. Not sure how relevant is it, just saying

Reply:
For now we need only pre-build scripts I guess.